src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -0,0 +1,130 @@
+import classNames from 'classnames';
+import { useConfig } from 'wagmi';
+import { Avatar, AvatarIcon, IconType, NumberFormat, formatterUtils } from '../../../../core';
+import { type ICompositeAddress, type IWeb3ComponentProps } from '../../../types';
+import { AssetTransferAddress } from './assetTransferAddress';
+
+export interface IRequiredCompositeAddress extends ICompositeAddress {
+    /**
+     * Required address the transfer participants.
+     */
+    address: string;
+}
+
+export interface IAssetTransferProps extends IWeb3ComponentProps {
+    /**
+     * Address of the transaction sender.
+     */
+    sender: IRequiredCompositeAddress;
+    /**
+     * Address of the transaction recipient.
+     */
+    recipient: IRequiredCompositeAddress;
+    /**
+     * Name of the token transferred.
```

Comment:
Let's update the comments as well
```suggestion
     * Name of the asset transferred.
```